manpages/Makefile

```diff
@@ -8,7 +8,7 @@ default:
 # 	./latex2man -t ./macros.trans $< $@
 
 %.1.gz: %.1
-	gzip --keep $<
+	gzip < $< > $<.gz
```

Comment:
What is this for?

Reply:
the `--keep` flag does not exist on older versions of `gzip` that are being used on CentOS. i was looking around online, and updating the version of `gzip` was recommended against. the code is functionally equivalent.